wondrous-app/services/apollo.ts

```diff
@@ -15,11 +15,13 @@ const httpLink = new HttpLink({
 });
 
 const getAuth = () => {
+  if (typeof window === 'undefined') return null;
```

Comment:
does this happen?

Reply:
This is for getServerSideProps function in task/[taskId]/nft, will never happen when fetching from the client side. We need this check because when we fetch getTaskMintTokenData in task/[taskId]/nft we don't have a localStorage

Reply:
make sense! forgot about that